store/read_transaction.cpp

```diff
@@ -266,6 +266,8 @@ struct store::results_iterator::impl : utils::noncopyable {
     bool _valid;
 
     /// Constructor.
+    ///
+    /// \param backend_ the store backend implementation.
```

Comment:
nit: Capital letter at the beginning of sentence. I see a few more instances of this; please review.

Reply:
Done. Ugh, this is annoying...